dockerStart.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# Little startup script fetches data dictionary, and runs the relay
+# compiler, then webpack before launching nginx
```

Comment:
can you put the time to run the script here?

Reply:
this pr need to be merged together with a cloud-automation pr

Reply:
added related pr in cloud-automation